### PR TITLE
Simplify Bezier, improve details and fix minor typos

### PR DESCRIPTION
**Pull request by CSharperMantle**
This PR proposes several improvements that enhance icons' symmetry, details, and simplicity, together with several minor fixes for typos.

The new SVG paths proposed are created solely by me, and I confirm and agree that my proposed code be licensed under MIT (the same as this repository), thus no code ownership or licensing issues should exist.

Many ❤️s again for this project you created!

### CODE REVIEW

**Comment by cchampou** [comment]
Thank you for this contribution this is awesome, will check this out, review and test it !

**Comment by CSharperMantle** [comment]
Glad to hear this! :wink: Please feel free to ask me should you have questions.

File: src/icons/anemo.jsx [hunk 1]
```diff
@@ -2,21 +2,8 @@ import React from 'react';
...
...m 9.912,2.429 C 9.056,7.189 7.663,7.826 7.737,9.049 a 0.973,0.973 0 0 0 1.1,0.984 c 0.316,0.013 0.724,-0.546 0.294,-0.612 -1.621,0 -1.022,-2.1 0.346,-1.4 a 5.222,5.222 0 0 0 2.343,0.687 c 1.8,-0.177 2.572,-2.3 1.989,-3.859 -0.871,1.303 -1.957,2.597 -3.697,2.414 z M 7.692,6.491 A 7.516,7.516 0 0 0 9.918,5.63 3.067,3.067 0 0 0 9.286,0.018 5.143,5.143 0 0 1 7.692,6.491 Z M 6.349,6.477 A 5.071,5.071 0 0 1 4.734,0 3.075,3.075 0 0 0 3.612,5.287 7.944,7.944 0 0 0 6.312,6.522 Z M 11.691,6.388 C 10.548,7.044 9.097,6.751 8.04,7.605 a 1.557,1.557 0 0 0 0.07,2.768 c 0.234,0.1 0.462,0.206 0.689,0.014 0.223,-0.167 0.4,-0.162 0.721,-0.012 a 9.094,9.094 0 0 0 -2.513,3.482 9.217,9.217 0 0 0 -2.523,-3.478 0.605,0.605 0 0 1 0.726,0 c 0.6,0.347 1.443,-0.4 1.555,-1 C 7.012,8.2 5.829,7.273 4.783,7.049 4.271,6.928 3.745,6.867 3.233,6.749 0.866,6.266 0.51,3.868 1.669,2.034 c 0.668,4.9 4,3.555 5.332,6.26 1.321,-2.671 4.678,-1.382 5.345,-6.261 0.811,1.416 1.054,3.523 -0.655,4.355 z M 7.571,11.579 7,11.2 6.428,11.585 7.008,12.557 Z M 7,10.066 a 1.592,1.592 0 0 1 -1,0.911 0.925,0.925 0 0 0 0.991,-0.272 c 0.426,0.327 0.794,0.417 1.046,0.255 A 1.561,1.561 0 0 1 7,10.066 Z" fill={color} style="fill-opacity:1" />
```
Comment (cchampou) [inline src/icons/anemo.jsx:6]:
```suggestion
    <path d="M 0.2,4.834 C 0.964,6.034 2.013,7.309 3.562,7.268 4.587,7.201 5.936,7.492 6.241,8.627 6.554,9.492 5.416,10.609 4.728,9.751 4.62,9.564 4.689,9.451 4.901,9.425 c 1,0.021 1.348,-1.12 0.4,-1.513 C 4.488,7.812 3.753,8.439 2.972,8.619 0.791,9.247 -0.524,6.656 0.2,4.834 Z m 9.912,2.429 C 9.056,7.189 7.663,7.826 7.737,9.049 a 0.973,0.973 0 0 0 1.1,0.984 c 0.316,0.013 0.724,-0.546 0.294,-0.612 -1.621,0 -1.022,-2.1 0.346,-1.4 a 5.222,5.222 0 0 0 2.343,0.687 c 1.8,-0.177 2.572,-2.3 1.989,-3.859 -0.871,1.303 -1.957,2.597 -3.697,2.414 z M 7.692,6.491 A 7.516,7.516 0 0 0 9.918,5.63 3.067,3.067 0 0 0 9.286,0.018 5.143,5.143 0 0 1 7.692,6.491 Z M 6.349,6.477 A 5.071,5.071 0 0 1 4.734,0 3.075,3.075 0 0 0 3.612,5.287 7.944,7.944 0 0 0 6.312,6.522 Z M 11.691,6.388 C 10.548,7.044 9.097,6.751 8.04,7.605 a 1.557,1.557 0 0 0 0.07,2.768 c 0.234,0.1 0.462,0.206 0.689,0.014 0.223,-0.167 0.4,-0.162 0.721,-0.012 a 9.094,9.094 0 0 0 -2.513,3.482 9.217,9.217 0 0 0 -2.523,-3.478 0.605,0.605 0 0 1 0.726,0 c 0.6,0.347 1.443,-0.4 1.555,-1 C 7.012,8.2 5.829,7.273 4.783,7.049 4.271,6.928 3.745,6.867 3.233,6.749 0.866,6.266 0.51,3.868 1.669,2.034 c 0.668,4.9 4,3.555 5.332,6.26 1.321,-2.671 4.678,-1.382 5.345,-6.261 0.811,1.416 1.054,3.523 -0.655,4.355 z M 7.571,11.579 7,11.2 6.428,11.585 7.008,12.557 Z M 7,10.066 a 1.592,1.592 0 0 1 -1,0.911 0.925,0.925 0 0 0 0.991,-0.272 c 0.426,0.327 0.794,0.417 1.046,0.255 A 1.561,1.561 0 0 1 7,10.066 Z" fill={color} style={{ fillOpacity: 1 }} />
```
This should be written in a React / JSX style 😉. The transpile process will do the rest and transform it into a valid style attribute.

Comment (CSharperMantle) [reply src/icons/anemo.jsx:6]: Nice catch, thanks!

File: src/icons/electro.jsx
[the remark this answers is not in the document]
Comment (CSharperMantle) [reply src/icons/electro.jsx:1]: It does look abnormal... I will look into this as soon as I finish my current task.

**Review by cchampou — APPROVED** [review]
💯  perfectly fixed, thank you again !

**Comment by cchampou** [comment]
I'll publish a release today on NPM 😉

**Pull request merged**